stdlib/list.mli

```diff
@@ -191,10 +191,19 @@ val rev_map : ('a -> 'b) -> 'a list -> 'b list
 val filter_map : ('a -> 'b option) -> 'a list -> 'b list
 (** [filter_map f l] applies [f] to every element of [l], filters
     out the [None] elements and returns the list of the arguments of
-    the [Some] elements.
+    the [Some] elements. It preserves the order of the elements (by
+    means of an internal reversal). It is tail-recursive.
     @since 4.08.0
  *)
 
+val rev_filter_map : ('a -> 'b option) -> 'a list -> 'b list
+(** [rev_filter_map f l] gives the same result as
+    {!rev}[ (]{!filter_map}[ f l)], but is more efficient:
+    [filter_map f l] is implemented as [rev rev_filtermap f l].
```

Comment:
```suggestion
    [filter_map f l] is implemented as [rev (rev_filter_map f l)].
```

Reply:
I think it is even better to not describe implementation details that could change between version.